packages/paste-website/src/pages/components/status-badge/index.mdx

```diff
@@ -97,8 +98,16 @@ export const getStaticProps = async () => {
 
 ## About Status Badge
 
+The Status badge is an implementation of the the [Status Pattern](/patterns/status), pre-built for you to display and indicate the status of a process or connection in your product.
+
+It pairs Status level with color and icon to assist with visual design and accessbility.
+
 ### Accessibility
 
+The idea of the Status Badge is that helps you communicate a status in a more accessible way but automatically pairing an icon with a color that represents a specific status level. This ensure there are distinct visual difference between each status type.
+
+Care must still be taken to ensure the word used to describe the status in the badge is unique to the status level it is paired with for any given screen.
+
 ## Examples
 
 ### Variants
```

Comment:
suggestion(non-blocking): Group all process types together in one example, and all connectivity types together in one example